assets/email_template.html

```diff
@@ -1,53 +1,134 @@
 <html>
```

Comment:
Ignore this - I forgot to have prettier ignore this and it modified a bunch of things that aren't important

Reply:
I think it's good that prettier fixes these actually!